### PR TITLE
Pia tool section 3

### PR DESCRIPTION
**Pull request by jbrou015**

### CODE REVIEW

File: ComplaintFormCore/ClientApp/src/surveyFile.ts [hunk 1]
```diff
@@ -197,22 +199,27 @@ export function updateFilePreview(survey: Survey.SurveyModel, question: Survey.Q
...
 
-            div.appendChild(buttonRemove);
+                buttonRemove.onclick = () => {
+                    if (confirm(getTranslation(question.confirmRemoveMessage, survey.locale))) {
+                        question.removeFile({ name: fileItem.name });
+                    }
+                };
+
+                div.appendChild(buttonRemove);
+            }
```
Comment (joshcampbell191) [inline ComplaintFormCore/ClientApp/src/surveyFile.ts:222]: Are we using the File field for this? If so, why aren't we using the [needConfirmRemoveFile](https://surveyjs.io/Documentation/Library?id=questionfilemodel#needConfirmRemoveFile) property to confirm file deletions?

Comment (jbrou015) [reply ComplaintFormCore/ClientApp/src/surveyFile.ts:222]: We are build our own File Preview so needConfirmRemoveFile is useless. I will remove this property from the json.

File: ComplaintFormCore/ClientApp/src/surveyFile.ts [hunk 2]
```diff
@@ -197,22 +199,33 @@ export function updateFilePreview(survey: Survey.SurveyModel, question: Survey.Q
...
+                buttonRemove.setAttribute("type", "button");
+                buttonRemove.className = "btn sv_q_file_remove_button";
+
+                if (question.itemListRemoveText && question.itemListRemoveText.length > 0) {
+                    buttonRemove.innerText = getTranslation(question.itemListRemoveText, surveyObj.locale);
+                } else if (Survey.surveyLocalization.locales[surveyObj.locale]) {
+                    //  This is strange but if the survey is in english
+                    //  Survey.surveyLocalization.locales[surveyObj.locale] is undefined.
+                    //  If in French then Survey.surveyLocalization.locales[surveyObj.locale] has values.
+                    buttonRemove.innerText = Survey.surveyLocalization.locales[surveyObj.locale].removeRow;
```
Comment (joshcampbell191) [inline ComplaintFormCore/ClientApp/src/surveyFile.ts:215]: What's the difference between getTranslation and Survey.surveyLocalization.locales[...]?

Comment (jbrou015) [reply ComplaintFormCore/ClientApp/src/surveyFile.ts:215]: getTranslation translate a property on a question and Survey.surveyLocalization.locales[surveyObj.locale] gets a translated value from the array of survey local values.

**Review by joshcampbell191 — APPROVED** [review]

**Pull request merged**